docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -79,4 +81,11 @@ See [upgrade notes]({{< relref "../installation/upgrading.md" >}}).
 
 Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md) for a complete list of new features, changes, and bug fixes.
 
+# News from other parts of the Grafana ecosystem
```

Comment:
can we remove this heading unless we have a text for this (headings on heads look bad).  and/or make this h2 and the the ADX heading should be h3 (if we keep this. heading)

Reply:
Yes! It is up to you! I wasn't sure how I should add it so this is just a suggestion. I will change it according to your thoughts so let me know :)